disnake/message.py

```diff
@@ -659,7 +659,8 @@ class InteractionReference:
     type: :class:`InteractionType`
         The type of interaction.
     name: :class:`str`
-        The name of the application command.
+        The full name of the application command, including group and subcommand name if applicable
+        (separated by spaces).
```

Comment:
might want to mention something about how for older messages, (created before july 18th) they may not include the full command

Reply:
Apparently this change [seems to be retroactive](https://canary.discord.com/channels/613425648685547541/697489244649816084/998690611198701568), will update

Reply:
Nope, this change is not retroactive. @shiftinv would you please update the text?

Reply:
https://github.com/DisnakeDev/disnake/pull/625/commits/6ccf52e31c603ca70505372b31d4ca82968afa63